straxen/plugins/nveto_recorder.py

```diff
@@ -90,14 +94,27 @@ def compute(self, raw_records_nv, start, end):
                     'lone_raw_records_nv': lr,
                     'lone_raw_record_statistics_nv': lrs}
 
+        # Search for hits to define coincidence intervals:
+        temp_records = strax.raw_to_records(raw_records_nv)
+        temp_records = strax.sort_by_time(temp_records)
+        strax.zero_out_of_bounds(temp_records)
+        strax.baseline(temp_records,
+                       baseline_samples=self.config['baseline_samples_nv'],
+                       flip=True)
+        hits = strax.find_hits(temp_records,
+                               min_amplitude=self.config['hit_min_amplitude_nv'])
+        del temp_records
```

Comment:
One thing I'm wondering, if we are already doing the hitfinding here, why not skip the nVETOPulseProcessing plugin and do those steps here as well? Effectively, you are now computing the same thing twice?

Reply:
I thought the same too, and I mostly agree. The two thing which are not applied yet are baselining and zero out of bounds. For the moment I would like to keep the possibility to go back to the raw data as we are still testing things.